README.md

```diff
@@ -24,6 +24,23 @@ yarn kafka:start
 yarn start
 ```
 
+## Project Structure
+
+To understand the overall architecture of the project, please review our [architecture documentation](docs/ARCHITECTURE.md).
+
+The root structure is as follows:
+
+```
+- docs     // Project documentation
+- services // Convenience containers for various external services (e.g. Kafka)
```

Comment:
`/services` confuses me a little bit, but I think it'll make sense as more stuff develops.

Reply:
The logic here is those really are (containerized) *services* -- kafka, postgres.

Maybe `/serviceContainers` is a better name, but I also didn't want to assume everything will be in a container necessarily.

We can definitely tweak over time.  The stuff in that folder is not going to be directly referenced by anything in `src` so it should be easy to modify.